hillrange/ckeditor/1.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Hillrange\\CKEditor\\HillrangeCKEditorBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "ckeditor:install --clear=drop": "symfony-cmd"
```

Comment:
This will be executed for each composer install/update, is it what you need?

Reply:
Thanks Fabian, Will change the ckeditor:install to skip.  Good Pickup.